src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -415,16 +415,16 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             _cursorTimer = std::make_optional(DispatcherTimer());
             _cursorTimer.value().Interval(std::chrono::milliseconds(blinkTime));
             _cursorTimer.value().Tick({ this, &TermControl::_BlinkCursor });
-
-            _controlRoot.GotFocus({ this, &TermControl::_GotFocusHandler });
-            _controlRoot.LostFocus({ this, &TermControl::_LostFocusHandler });
         }
         else
         {
             // The user has disabled cursor blinking
             _cursorTimer = std::nullopt;
         }
 
+        _controlRoot.GotFocus({ this, &TermControl::_GotFocusHandler });

```

Comment:
I'd want to keep this change even if we can use `_controlRoot.FocusState` (as Carlos suggested)